vignettes/compatibility.Rmd

```diff
@@ -0,0 +1,70 @@
+---
+title: "Compatibility"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Compatibility}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+R7 is designed to be compatible with S3 and S4. This vignette discusses the details.
+
+```{r setup}
+library(R7)
+```
+
+## S3
```

Comment:
Probably worth describing how R7 is just an extension of S3 (objects have a class vector), so that a lot of this "just works".